runtime/tutor/en/vim-01-beginner.tutor

```diff
@@ -582,14 +582,14 @@ NOTE: You can also select the lines you want to substitute first using visual-mo
     `<C-o>`{normal} takes you back to older positions, `<C-i>`{normal} to
     newer positions.
 
- 3. Typing `%`{normal} while the cursor is on a (,),[,],{, or } goes to its
+ 3. Typing `%`{normal} while the cursor is on a (, ), [, ], {, or } goes to its
     match.
 
  4. To substitute new for the first old in a line type
 ~~~ cmd
         :s/old/new
 ~~~
-    To substitute new for all 'old's on a line type
+    To substitute new for all olds on a line type
```

Comment:
This is a bit strange

Reply:
As per the other thread, these changes are explained in more detail here: https://github.com/neovim/neovim/pull/27689

Reply:
Specifically for this one: https://github.com/neovim/neovim/commit/c736321075e76708542d01edede38abfd4d6faa4

Reply:
thanks